cli/jobs/pipelines/automl/image-instance-segmentation-task-fridge-items-pipeline/pipeline.yml

```diff
@@ -0,0 +1,59 @@
+$schema: https://azuremlschemas.azureedge.net/latest/pipelineJob.schema.json
+type: pipeline
+
+description: Pipeline using AutoML Image Instance Segmentation task
+
+display_name: pipeline-with-image-instance-segmentation
+experiment_name: pipeline-with-automl
+
+settings:
+  default_compute: azureml:gpu-cluster
+
+inputs:
+  image_instance_segmentation_training_data:
+    type: mltable
+    path: data/training-mltable-folder
+  image_instance_segmentation_validation_data:
+    type: mltable
+    path: data/validation-mltable-folder
+
+jobs:
+  image_instance_segmentation_node:
+    type: automl
+    task: image_instance_segmentation
+    log_verbosity: info
+    primary_metric: mean_average_precision
+    limits:
+      timeout_minutes: 180
+    target_column_name: label
+    training_data: ${{parent.inputs.image_instance_segmentation_training_data}}
+    validation_data: ${{parent.inputs.image_instance_segmentation_validation_data}}
+    image_model:
+      early_stopping: True
+      evaluation_frequency: 1
+    sweep:
+      limits:
+        max_trials: 10
+        max_concurrent_trials: 2
+      sampling_algorithm: random
+      early_termination:
+        type: bandit
+        evaluation_interval: 2
+        slack_factor: 0.2
+        delay_evaluation: 6
+    search_space:
+      - model_name: "maskrcnn_resnet50_fpn"
```

Comment:
This search space format is going to be changed in the September release of sdk v2. Please sync with Nilesh to get the correct format for these.

Reply:
Can we can check this as part of separate PR?

Reply:
not sure if the plan is to merge this before or after the release of sdk v2.. but these examples will start failing once the new sdk release is out...

Reply:
@sharma-riti Planning to have that as a separate change. Current workflow will start to fail if we add it now